integr8ly/integreatly-operator
# Mgdapi 1595 cluster scoped 3scale

# Issue link

MGDAPI-1595

# What

feature branch merge

# Verification steps
- install rhoam 1.24.1 by applying the catalog source and installing rhoam from the operator hub
```bash
oc apply -f - <<EOF              
---
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: rhmi-operators
  namespace: openshift-marketplace
spec:
  image: 'quay.io/austincunningham/managed-api-service-index:1.24.1'
  sourceType: grpc
EOF

```
- once the install is finished confirm that the 3scale operator is namespaces scoped (Managed Namespaces) viewing install operators on cluster
![image](https://user-images.githubusercontent.com/16667688/180149194-9f1684d8-7826-4f3d-a3f9-b046e9e534e3.png)

- update the catalog source image to point to `quay.io/austincunningham/managed-api-service-index:1.25.0` in the ui or via oc edit
```bash
oc edit catalogsource rhmi-operators --namespace openshift-marketplace
```
- wait a few minuets and approve the upgrade for rhoam in installed operators
- once the upgrade is complete you should see the 3scale operator switch to cluster scoped (Managed Namespaces) 
![image](https://user-images.githubusercontent.com/16667688/180152240-a3923c38-f75c-46de-9ac6-16f88d449297.png)
